src/algebra/direct_sum/internal.lean

```diff
@@ -77,6 +77,24 @@ direct_sum.to_semiring (λ i, (A i).subtype) rfl (λ _ _ _ _, rfl)
   direct_sum.submonoid_coe_ring_hom A (direct_sum.of (λ i, A i) i x) = x :=
 direct_sum.to_semiring_of _ _ _ _ _
 
+section big_operators
+
+open_locale big_operators
```

Comment:
I think it's fine to just do this at the top of the file, and then get rid of the `section`